addons/pos_event/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+{
+    'name': 'Point of Sale Events',
+    'version': '1.0',
+    'category': 'Sales/Point of Sale',
+    'description': """
+Creating registration through the Point of Sale.
+================================================
+This module allows you to automate and connect your registration creation with
+your main point of sale flow.
+It defines a new kind of service products that offers you the possibility to
+choose an event category associated with it. When you encode a pos order for
+that product, you will be able to choose an existing event of that category and
+when you validate your pos order it will automatically create a registration for
+this event.
+""",
+    'depends': ['event', 'point_of_sale', 'event_product'],
+    'data': [
+        'data/pos_event_data.xml',
+        'report/event_product_report_views.xml',
+        'views/event_event_views.xml',
+    ],
+    'installable': True,
+    'auto_install': ['event', 'point_of_sale', 'event_product'],
```

Comment:
You can put auto_install: True as your condition is the same as the dependencies